horovod/runner/elastic/discovery.py

```diff
@@ -123,7 +188,7 @@ def current_hosts(self):
 
     def blacklist(self, host):
         if not self._hosts_state[host].is_blacklisted():
-            logging.warning('blacklist failing host: {}'.format(host))
+            logging.debug('blacklist failing host: {}'.format(host))
```

Comment:
I think this is useful in  normal operation, so what about moving this to `info` level?

Reply:
Done